[2024-03-02T03:16:45.560+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ecommerce_pipeline.merge_data scheduled__2024-03-02T02:01:00+00:00 [queued]>
[2024-03-02T03:16:45.578+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ecommerce_pipeline.merge_data scheduled__2024-03-02T02:01:00+00:00 [queued]>
[2024-03-02T03:16:45.579+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-03-02T03:16:45.597+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): merge_data> on 2024-03-02 02:01:00+00:00
[2024-03-02T03:16:45.603+0000] {standard_task_runner.py:60} INFO - Started process 1847 to run task
[2024-03-02T03:16:45.609+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'ecommerce_pipeline', 'merge_data', 'scheduled__2024-03-02T02:01:00+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/ecom.py', '--cfg-path', '/tmp/tmpsydxpzso']
[2024-03-02T03:16:45.617+0000] {standard_task_runner.py:88} INFO - Job 63: Subtask merge_data
[2024-03-02T03:16:45.640+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-02T03:16:45.690+0000] {task_command.py:423} INFO - Running <TaskInstance: ecommerce_pipeline.merge_data scheduled__2024-03-02T02:01:00+00:00 [running]> on host 70b88be0f8e4
[2024-03-02T03:16:46.034+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ecommerce_pipeline' AIRFLOW_CTX_TASK_ID='merge_data' AIRFLOW_CTX_EXECUTION_DATE='2024-03-02T02:01:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-02T02:01:00+00:00'
[2024-03-02T03:16:46.036+0000] {logging_mixin.py:188} INFO - Error: Can only merge Series or DataFrame objects, a <class 'str'> was passed
[2024-03-02T03:16:46.036+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-03-02T03:16:46.048+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=ecommerce_pipeline, task_id=merge_data, execution_date=20240302T020100, start_date=20240302T031645, end_date=20240302T031646
[2024-03-02T03:16:46.101+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-02T03:16:46.135+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
